### ISSUE TITLE
Points selected in table persisting after points de-selected

### ISSUE THREAD

**Issue by AmberSJones** [ODM2/ODMToolsPython#218]
If I apply a filter, clear it, and apply another filter, the points selected with the original filter are persisting as selected in the data table. They are not selected in the plot, and if I perform an action (e.g., interpolate), they are not included. Points appear to be selected in the table even after I use the 'Reset Selection'.

![image](https://cloud.githubusercontent.com/assets/5023186/6985514/820c4e9e-d9f1-11e4-8d11-d036e2766e5d.png)

**Comment by jmeline** [comment]
@AmberSJones

**Comment by AmberSJones** [comment]
Ok, this is functioning correctly when the filter is cleared, however, if I try to filter from the previous filter, all of the points from the original filter remain selected in the table. In the screenshot below, I used the filter to select a date range. I then used the filter from previous to select anything in that date range >7. The correct points are selected on the plot, but the original selection persists in the table.
![image](https://cloud.githubusercontent.com/assets/5023186/7358195/204c7dce-ecf1-11e4-90e8-29582c17251a.png)
